tests/compute/credentials.go

```diff
@@ -117,6 +117,40 @@ var _ = SIGDescribe("Guest Access Credentials", func() {
 		),
 	)
 
+	It("[test_id:TODO] should remove authorized_keys file if secret is empty", func() {
```

Comment:
The secret is not empty, but the length of the key is zero. Is that what you mean?

Reply:
What happens if the data of the Secret is empty, i.e. not entries?

Reply:
I've removed the empty key.